setup.py

```diff
@@ -87,6 +87,7 @@
         "fairscale",
         "transformers",
         "gtn==0.0.0",
+        "evaluate",
```

Comment:
can you add the version that worked for you?

Reply:
I think the evaluate codebase is updated with transformers, so maybe getting the latest version like transformers is best ?